File: Data Structures & Algorithms/max-water-container/submission-0.java
class Solution {
    public int maxArea(int[] heights) {
        int maxArea=0;
        for(int i=0; i< heights.length; i++){

            int minLength=0;
            for( int j=i+1; j< heights.length; j++){
             
             minLength=Math.min(heights[i], heights[j] );
             
             int area= minLength*(j-i);

             maxArea = Math.max(maxArea, area);

            }
        }

     return maxArea;   
    }
}

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.cs
public class Solution {
    public int MaxProfit(int[] prices) {
        int maxProfit = 0;
        int min= prices[0];
        for(int i =0; i<prices.Length; i++)
        {
            
            min = Math.Min(min, prices[i]);
            maxProfit = Math.Max(maxProfit, (prices[i]- min));
        }
        return maxProfit;
    }
}
